components/NavBar/index.tsx

```diff
@@ -31,23 +30,37 @@ const Navbar = () => {
 
   return (
     <header className='fixed inset-x-0 top-0 z-10 py-3 bg-primary'>
-      <div className="container flex justify-between px-3 mx-auto">
+      <div className='container flex justify-between px-3 mx-auto'>
         <Link href='/'>
-          <a className='px-4 py-1 font-serif text-xl text-white hover:opacity-75'>
-            Home
+          <a className='hover:opacity-75'>
+            <Image
+              className='w-full h-full'
+              src={logo}
+              alt='Foodbank logo'
+              placeholder='blur'
+            />
           </a>
         </Link>
-        <nav className='flex items-center space-x-10'>
+        <nav className='flex items-center space-x-10' role='navigation'>
           {links.map(nav => (
             <NavLink key={nav.page} {...nav} />
           ))}
         </nav>
-        <button
-          className='px-4 py-1 font-serif text-xl text-white hover:opacity-75'
-          onClick={toggleSignIn}
-        >
-          Sign-in
-        </button>
+        {signIn ? (
+          <button
+            className='px-4 py-1 font-serif text-xl text-white hover:opacity-75'
+            onClick={toggleSignIn}
+          >
+            SIGN-OUT
+          </button>
+        ) : (
+          <button
+            className='px-4 py-1 font-serif text-xl text-white hover:opacity-75'
+            onClick={toggleSignIn}
+          >
+            SIGN-IN
+          </button>
+        )}
```

Comment:
Rather than re-rendering the entire button, it would be easier just to change the text inside